src/test/java/org/kohsuke/github/PayloadRule.java

```diff
@@ -0,0 +1,87 @@
+package org.kohsuke.github;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import org.apache.commons.io.IOUtils;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * @author Stephen Connolly
+ */
+public class PayloadRule implements TestRule {
```

Comment:
This is a kohsuke project not a jenkins project (would be nice if it was something we could release without KK)